Makefile

```diff
@@ -13,6 +13,7 @@ ifeq ($(OFFICIAL),yes)
     RELEASE ?= 1
 else
     ifeq ($(origin RELEASE), undefined)
+		VERSION = $(shell git describe --tags)
```

Comment:
This needs to be 1 tab followed by the proper number of spaces. See line below as reference.